src/minMaxLength.js

```diff
@@ -26,10 +26,10 @@ const minMaxLength = (options) => {
         return new Promise((resolve, reject) => {
             if (field.inputEls) {
                 resolve({
-                    valid: field.inputEls.some(el => el.length === 0 || (el.length > 0 && meetsMin(el) && meetsMax(el)))
+                    valid: field.inputEls.some(el => el.value.length === 0 || (el.value.length > 0 && meetsMin(el) && meetsMax(el)))
```

Comment:
Correct me if I'm wrong, but I think you can get rid of `el.value.length > 0`. Since that's an OR statement, it'll only hit the second part if value.length !== 0